src/PresentationalComponents/SystemsTable/SystemsTable.js

```diff
@@ -87,10 +93,18 @@ const SystemsTable = () => {
       values.length > 1 &&
       values.includes('yes') &&
       values.shift();
-    values.length > 0
-      ? setFilters({ ...filters, offset: 0, ...{ [param]: values } })
+    const passValue =
+      param === SFC.rhel_version.urlParam
+        ? Object.values(values || {}).flatMap((majorOsVersion) =>
```

Comment:
values is treated as an arrayabove (e.g 93th line), while you are treating as an object. I suspect it is intended right?

Reply:
@mkholjuraev yeah, this is intended for the group filters (here the rhel version), since it returns it's value as an object like 
```js
{8.0: {8.0: true, 8.1: true,...},...}
```

But the api/filter functions expect an array.